CommandHelper/LocalPackages/commands/chars/newtscamander.yml

```diff
@@ -6,8 +6,8 @@ costumes:
   val: eyJ0aW1lc3RhbXAiOjE1MjEwNTIwMTkxMzcsInByb2ZpbGVJZCI6IjkwYjJiYTZkNDNlNjRlNWM4Zjc3OTEyNmI2OTY5YTljIiwicHJvZmlsZU5hbWUiOiJXaW5nQXJtb3IiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzgxODRkOTFhNTg5ZGY5ZmFlOGZiMzhjYTc4YmRjY2E3ZjIwNGFjNDgyYjE3NzJjOWE3MTdjNGNmNTc2MWIxZSJ9fX0=
 effects: [
   ]
-items: [
-  ]
+items: 
```

Comment:
you deleted the empty brackets, and now you've assigned nothing to this line. this will not compile, sry brahhhhhhh